### DISCUSSION TITLE
Unminified version missing

### DISCUSSION THREAD

**Discussion by manake** [josdejong/mathjs#2781] [Q&A]
4 related issues to the general download/setup sequence:

1. Unminified version is not present on the downloads page.
2. /dist/ should contain math.js, math.min.js, math.esm.js, not sure why it doesn't. This is what most other repos do. /lib/browser/ is less good so I'd suggest abandoning this path altogether.
3. The ESM download is missing (can't be downloaded from CDN easily). No installation instructions for ESM are present but I assume it's `import * as mathjs from ./lib/esm/index.js`.
4. /lib/browser/ doesn't exist in the GitHub repo (but exists on NPM) so you can't run the examples by cloning the repo without building.

**Comment by josdejong** [comment]
1. You can either use the minified bundle, the non-minified ES module source code, or the non-minified CommonJS code. The minified bundle comes with a source map. 
2. The output code has been moved from `/dist` to `/lib`. The dist folder is only there for backward compatibility, to give you a clear error message with instruction.
3. The ESM files are not missing, they are located in the folder `/lib/esm`. In most JavaScript projects, there is no need to manually point to a specific index.js version, that is automatically selected based on whether you're in an ESM project or CommonJS. The installation instructions can be found here: https://mathjs.org/docs/getting_started.html, which is one of the main pages of the website.
4. That is correct. The source code repo only contains source code and no build outputs. To get an example running, you can for example copy the code somewhere and replace the import like `const math = require('..')` with something like `import math from 'mathjs'`. We could improve on that. It would be nice for example to have a link [in the examples ](https://mathjs.org/examples/index.html) to open the example in CodeSandbox or jsbin or jsfiddle or so, so you can directly play around with it.
